scripts/release.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -eu
+set -o pipefail
+
+echo -n "Build timestamp: "
+TIMESTAMP=$(date +%s)
+echo $TIMESTAMP
+
+echo -n "Git commit: "
+if [[ "$(git diff --stat)" != '' ]] || [[ -n "$(git status -s)" ]]; then
+  COMMIT=$(git rev-parse HEAD)-dirty
+  RELEASE=false
+else
+  COMMIT=$(git rev-parse HEAD)
+  RELEASE=true
+fi
+echo $COMMIT
+
+echo -n "Tagged version: "
+if git describe --tags --exact-match --match "v[0-9]*.[0-9]*.[0-9]*"; then
+  VERSION=$(git describe --tags --exact-match --match "v[0-9]*.[0-9]*.[0-9]*")
+  echo $VERSION
+else
+  VERSION=v0.0.0
+  RELEASE=false
+fi
+
+echo Running "go $@"
+exec go "$1" -ldflags \
```

Comment:
We definitely should start using buildinfo instead of release arguments, it's less code to maintain. But, I guess fine for now.

Reply:
I support that effort, but in favor of getting this out, we should proceed as is :+1: